Source/WebKit/NetworkProcess/Downloads/Download.h

```diff
@@ -86,6 +86,10 @@ class Download : public IPC::MessageSender, public CanMakeWeakPtr<Download> {
     void publishProgress(const URL&, SandboxExtension::Handle&&);
 #endif
 
+#if HAVE(MODERN_DOWNLOADPROGRESS)
+    void publishProgress(const URL&, std::span<const uint8_t>&&);
```

Comment:
You don't need to pass span by r-value references and use WTFMove(). spans are very cheap to copy.